### PR TITLE
Add ulauncher-toggle query param to set up initial query.

### PR DESCRIPTION
**Pull request by ajboni**
This PR makes possible to pre set the query when calling `ulauncher-toggle` 
usage: 

`ulauncher-toggle` => Same  default behavior
`ulauncher-toggle -q some_query`
`ulauncher-toggle -q "some long query"`
`ulauncher-toggle --query=some_query`
`ulauncher-toggle --query="some long query"`

### CODE REVIEW

File: ulauncher/main.py [hunk 1]
```diff
@@ -39,6 +39,11 @@ def setup(self, _):
         self.window.set_application(self)
         self.window.finish_initializing()
 
+    def do_before_emit(self, data ):
+        query = data.lookup_value("query",  GLib.VariantType("s"))
+        if query:      
+           self.window.initial_query = query.unpack() + " "
```
Comment (friday) [inline ulauncher/main.py:45]: I think we shouldn't add the space. If people want the space they can send ulauncher-toggle -q "keyword ". If people don't want a space they can't send negative one character.

**Review by friday — CHANGES_REQUESTED** [review]
Looks good and simple.
1. I don't think we should pad with the space.
2. I don't know if I think we *need* the getopts logic rather than just `$1`, but it's more future-proof if we do eventually want more arguments, so I'm good with that.

File: ulauncher/main.py (hunk 1 above)
Comment (ajboni) [reply ulauncher/main.py]: Yes, makes sense!

**Review by friday — APPROVED** [review]

**Comment by friday** [comment]
Thanks again :+1:

**Pull request merged**